release-notes/09-2022/README.md

```diff
@@ -0,0 +1,169 @@
+# SystemLink Enterprise 9-2022 Release Notes
+
+The 09-2022 release bundle for SystemLink Enterprise has been published to <https://niedge01.jfrog.io>. Work with your account representative to obtain credentials to access these artifacts.
+
+**Note:**  If you are upgrading from the 07-2022 release or earlier there is a potential for file meta data loss. [FileIngestionDatabaseRecovery.md](/FileIngestionDatabaseRecovery.md) includes a procedure migrate this data through the upgrade.
```

Comment:
```suggestion
**Note:**  If you are upgrading from the 07-2022 release or earlier. Complete the steps in [FileIngestionDatabaseRecovery.md](/FileIngestionDatabaseRecovery.md) to avoid losing file metadata.
```